### PR TITLE
Added example of Java SDK

### PR DESCRIPTION
**Pull request by nksprzak**

### CODE REVIEW

File: examples/example-app/pom.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          <groupId>com.iland.core</groupId>
+          <artifactId>iland-sdk</artifactId>
+          <version>0.9.2</version>
+      </dependency>
+  </dependencies>
+  <repositories>
+      <repository>
+          <id>artifacts.iland.dev</id>
+          <name>iland artifacts repository</name>
+          <url>http://artifacts.iland.dev/nexus/content/groups/public</url>
```
Comment (bsnyder788) [inline examples/example-app/pom.xml]: this stuff should not be in here.

Comment (bsnyder788) [reply examples/example-app/pom.xml]: ...you need to use the one on github.

File: examples/example-app/src/main/resources/app.properties [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# base url of the iland console
+base.url=https://api.ilandcloud.com/ecs
+# access url of the iland console
+access.url=https://console.ilandcloud.com/auth/realms/iland-core/
```
Comment (bsnyder788) [inline examples/example-app/src/main/resources/app.properties]: the access url should be: https://console.ilandcloud.com/auth/realms/iland-core/protocol/openid-connect/token

File: examples/example-app/src/main/java/com/ilandjavasdk/app/App.java [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+    final User user = userResource.getUser(property.getUsername());
+    getUsersInventoryAndPrintAllEntities(user);
+    if (!catalogUuid.isEmpty() && !vdcUuid.isEmpty() && !vappTemplate.isEmpty()
+        && !networkUuid.isEmpty()) {
+      final String vappUuid = createVapp();
+      preformExampleVmOperations(vappUuid);
+      downloadVappAndUploadOva(vappUuid);
+    } else {
+      System.out.println(
+          "Could not get required uuids. Skipping everything and logging out.");
```
Comment (bsnyder788) [inline examples/example-app/src/main/java/com/ilandjavasdk/app/App.java]: UUIDs

File: examples/example-app/README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+### Build project
+```
```
Comment (bsnyder788) [inline examples/example-app/README.md:13]: Let's put a nice title and a few sentences about what the example does so they don't have to parse through code and comments to figure out.

File: examples/example-app/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Example of iland Cloud Java SDK
+In this example app, we use the Java SDK to perform various operations.<br />
+With the SDK, we login into the console. Then we get and print all the permissions for the user.
```
Comment (bsnyder788) [inline examples/example-app/README.md]: permissions?

File: examples/example-app/README.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Example of iland Cloud Java SDK
+In this example app, we use the Java SDK to perform various operations.<br />
```
Comment (bsnyder788) [inline examples/example-app/README.md]: why the <br /> tag?

File: examples/example-app/README.md (hunk 5 above)
Comment (cfsnyder) [reply examples/example-app/README.md]: I also wouldn't say that we log in to the console because this isn't related to the console in any way

**Review by bsnyder788 — APPROVED** [review]

**Pull request merged**